bin/package_debian

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
```

Comment:
Would be nice to move `bin/server_*` scripts in some manner that we dont loose Git history